apps/omg/lib/omg/crypto.ex

```diff
@@ -37,7 +37,7 @@ defmodule OMG.Crypto do
   @doc """
   Recovers address of signer from binary-encoded signature.
   """
-  @spec recover_address(hash_t(), sig_t()) :: {:ok, address_t()} | {:error, :signature_corrupt}
+  @spec recover_address(hash_t(), sig_t()) :: {:ok, address_t()} | {:error, :signature_corrupt | atom}
   def recover_address(<<digest::binary-size(32)>>, <<packed_signature::binary-size(65)>>) do
```

Comment:
hey @achiurizo my comment https://github.com/omisego/elixir-omg/pull/756#discussion_r289621724 here was not to say that binary() is wrong, but that dialyzer is unable to be specific what the contents of a binary are.
there's not much we can do because down the rabbit hole, thats how `:libsecp256k1.ecdsa_recover_compact/4` works